File: 코딩 기초 트레이닝/간단한 식 계산하기-p181865/간단한 식 계산하기.py
# https://school.programmers.co.kr/learn/courses/30/lessons/181865

def solution(binomial):
    answer = 0

    binomial = list(binomial.strip().split(" "))
    a = int(binomial[0])
    op = binomial[1]
    b = int(binomial[2])

    if op == '+':
        return a + b
    elif op == '-':
        return a - b
    else:
        return a * b

File: 코딩 기초 트레이닝/간단한 식 계산하기-p181865/간단한 식 계산하기.js
// https://school.programmers.co.kr/learn/courses/30/lessons/181865

function solution(binomial) {
    binomial = binomial.trim().split(" ");
    const a = Number(binomial[0])
    const op = binomial[1]
    const b = Number(binomial[2])

    if(op === '+') return a + b;
    if(op === '-') return a - b;
    return a * b;
}
